nucypher/blockchain/eth/actors.py

```diff
@@ -493,7 +493,7 @@ def start_staking_loop(self, now=True) -> None:
     @property
     def is_staking(self) -> bool:
         """Checks if this Miner currently has active stakes / locked tokens."""
-        return bool(self.locked_tokens > NU.ZERO())
+        return bool(self.stakes)
```

Comment:
Just curious: why `bool(self.stakes)` is the response here? What's the underlying logic?

Reply:
` self.__read_stakes()  # "load-in":  Read on-chain stakes` "syncs" the onchain stakes into the local staking cache on `Staker`, and will be set to `NO_STAKES.bool_value(False)` if there are no active stakes.